_data/sites.json

```diff
@@ -6191,6 +6192,15 @@
         "notes_tr": "\"Verilerinizin silinmesini talep edin\" tuşuna ve ardından \"Evet, eminim\" seçeneğine tıklayın. İşlemi bitirmek için bağlantı içeren bir e-posta alacaksınız.",
         "domains": ["etsy.com"]
     },
+    {
+        "name": "Europa Lehrmittel",
+        "url": "https://www.europa-lehrmittel.de/Datenschutz/",
+        "difficulty": "hard",
+        "notes": "Write an email to dsb@europa-lehrmittel.de.",
+        "notes_de": "Schicken sie eine E-Mail an dsb@europa-lehrmittel.de.",
```

Comment:
Since we have the email field already, you can drop it from the notes (applies to the other entries as well)